apps/condo/domains/resident/schema/RegisterResidentService.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generated by `createservice property.RegisterResidentService --type mutations`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('../access/RegisterResidentService')
+const { Resident } = require('../utils/serverSchema/index')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+
+const RegisterResidentService = new GQLCustomSchema('RegisterResidentService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterResidentInput { dv: Int!, sender: JSON!, address: String!, addressMeta: JSON!, unitName: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canRegisterResident,
+            schema: 'registerResident(data: RegisterResidentInput!): Resident',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                if (!context.authedItem.id) throw new Error('[error] User is not authenticated')
```

Comment:
You have checked this in access.canRegisterResident already

Reply:
Removed in `02d4dbaf`.